src/class.ziprecipes.php

```diff
@@ -69,10 +69,14 @@ public static function init()
 
 		closedir($pluginsDirHandle);
 
+        // time to init shortcode facade
+        $shortcodes = new __shortcode();
```

Comment:
nowhere. and i do not quite understand why it so. mybe it is because of namespace.
need to discuss